test/extended/prometheus/prometheus.go

```diff
@@ -49,6 +52,42 @@ type TelemeterClientConfig struct {
 	Enabled *bool `json:"enabled"`
 }
 
+var _ = g.Describe("[sig-instrumentation][Late] OpenShift service monitors [apigroup:image.openshift.io]", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("prometheus")
+
+	g.It("should not be accessible without authorization", func() {
+		var errs []error
+		client, err := prometheusoperatorv1client.NewForConfig(oc.AdminConfig())
+		if err != nil {
+			g.Fail(fmt.Sprintf("failed to create monitoring client: %v", err))
+		}
+		serviceMonitorList, err := client.ServiceMonitors("").List(context.Background(), metav1.ListOptions{})
```

Comment:
FYI: not sure if there is servicemonitor created and monitored by user-workload-monitoring, if so, we don't need to check for it. only check for servicemonitors under `openshift-.*` is fine

Reply:
Updated.